cavokeclientmodel.cpp

```diff
@@ -4,17 +4,14 @@
 
 
 CavokeClientModel::CavokeClientModel(QObject *parent)
-    : QObject{parent} {
+        : QObject{parent} {
 }
 
-void CavokeClientModel::loadQmlGame(const QString &gameName) {
-    // TODO: different games loading
-
+void CavokeClientModel::loadQmlGame(const QString &appPath) {
     // TODO: change to url loading from cache, not resources
-    // TODO: delete resource.qrc
-    auto* gameModel = new CavokeQmlGameModel (QUrl("qrc:/tictactoe-files/tic-tac-toe.qml"), this); // must be alive when accessed from QML
+    auto *gameModel = new CavokeQmlGameModel(appPath, this); // must be alive when accessed from QML

```

Comment:
```suggestion
    auto *gameModel = new CavokeQmlGameModel(QUrl::fromUserInput(appPath), this); // must be alive when accessed from QML
```
Без этого у меня почему-то совсем отказывалось работать. Выводились такие ошибки:
```
c:/Users/waleko/Documents/Projects/hse/cpp-exam/client-prototype/tictactoe-files/tic-tac-toe.qml: Network error
An error(s) occurred while opening the QML app: 

c:/Users/waleko/Documents/Projects/hse/cpp-exam/client-prototype/tictactoe-files/tic-tac-toe.qml: Network error
```

[qt-bugtracker](https://bugreports.qt.io/browse/QTBUG-85977?focusedCommentId=522384&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-522384)